cms/djangoapps/contentstore/views/preview.py

```diff
@@ -213,10 +214,25 @@ def _preview_module_system(request, descriptor, field_data):
             "i18n": ModuleI18nService,
             "settings": SettingsService(),
             "user": DjangoXBlockUserService(request.user),
+            "partitions": PreviewPartitionService(user=request.user, course_id=course_id)
         },
     )
 
 
+class PreviewPartitionService(PartitionService):
+    """
+    Another runtime mixin that provides access to the student partitions defined on the
```

Comment:
@nasthagiri These comments will be updated

Note that the implementation is now identical to to the LMSPartitionService. But it's so little code, that I don't think it matters.